README.md

```diff
@@ -1,7 +1,7 @@
 bedrock
 =======
 
-*bedrock* is the code name of the new mozilla.org. It is as shiny,
+*bedrock* is the code name of mozilla.org. It is as shiny,
```

Comment:
This is not exactly accurate yet. It's the code name for the django-based web site that has been gradually replacing the old site for years now. We're almost done, so maybe this will be accurate soon. But I think something more like "Bedrock is the name we use for the code behind www.mozilla.org" would be good.